plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlScannerTest.java

```diff
@@ -64,4 +96,20 @@ public void testStartElement_correctVersion() throws SAXException {
     assertEquals(0, scanner.getBlacklist().size());
   }
   
+  @Test
+  public void testFindClass() {
+    IType type = WebXmlScanner.findClass("ServletClass");
+    assertNotNull(type);
+  }
+  
+  private static void createFolders(IContainer parent, IPath path) throws CoreException {
```

Comment:
We already have similar methods in `com.google.cloud.tools.eclipse.appengine.newproject.CodeTemplates`. We could extract it into a util class and use it here instead of reimplementing. In a separate PR.